crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -612,6 +616,25 @@ type Epoch {
 	"""
 	protocolConfigs: ProtocolConfigs
 	"""
+	Information about whether last epoch change used safe mode, which happens if the full epoch
```

Comment:
Now that this is part of `Epoch` prefer phrasing this in terms of the current epoch, rather than the last epoch (i.e. the start of this epoch was in safe mode).

Reply:
Definitely, thanks for catching that!